### PR TITLE
[UFC-1051] Add docs for react client setting

### PR DESCRIPTION
**Pull request by WimJongeneel**
Can be released ASAP, the React client went into Beta in 10.7

### CODE REVIEW

File: content/en/docs/refguide/modeling/app-explorer/app/app-settings/_index.md [hunk 1]
```diff
@@ -27,19 +27,27 @@ For more information on settings in a configuration, see [Configuration](/refgui
 
 These settings influence the behavior of the Runtime when running your application.
 
-### 3.1 Static Resources from Disk
+### 3.1 Use React Client (Beta)
+
+This setting enables the new React client. This client was released into Beta with Mendix 10.7. There are three options:
+
+- _No_: Don't use the React (default).
+- _Yes_: Use the React client. In this mode you will get consistency errors for imcompatable widgets.
```
Comment (EnasAbdelrazek) [inline content/en/docs/refguide/modeling/app-explorer/app/app-settings/_index.md]:
```suggestion
- _Yes_: Use the React client. In this mode, you will get consistency errors for incompatible widgets.
```

File: content/en/docs/refguide/modeling/app-explorer/app/app-settings/_index.md [hunk 2]
```diff
@@ -27,19 +27,27 @@ For more information on settings in a configuration, see [Configuration](/refgui
...
+- _Migration mode_: Use the React client, but ignore incompatable widgets. Recommended for trying out the new client.
```
Comment (EnasAbdelrazek) [inline content/en/docs/refguide/modeling/app-explorer/app/app-settings/_index.md]:
```suggestion
- _Migration mode_: Use the React client and ignore incompatible widgets. Recommended for trying out the new client.
```

**Comment by EnasAbdelrazek** [comment]
@WimJongeneel Could you please add a screenshot for this setting?

**Comment by WimJongeneel** [comment]
@EnasAbdelrazek we could, but would that add anything? We don't have for the other 10 out of 11 settings in this panel, and I don't think a picture of the radio buttons with exact labels as described will explain anything father.

**Comment by EnasAbdelrazek** [comment]
@WimJongeneel I wanted to show that it is part of the runtime tab, but now I see that it is in the runtime tab section.

**Review by EnasAbdelrazek — APPROVED** [review]

File: content/en/docs/refguide/modeling/app-explorer/app/app-settings/_index.md [hunk 3]
```diff
@@ -27,19 +27,27 @@ For more information on settings in a configuration, see [Configuration](/refgui
...
 These settings influence the behavior of the Runtime when running your application.
 
-### 3.1 Static Resources from Disk
+### 3.1 Use React Client (Beta)
+
+This setting enables the new React client. This client was released into Beta with Mendix 10.7. There are three options:
+
+- _No_: Don't use the React (default).
+- _Yes_: Use the React client. In this mode, you will get consistency errors for incompatible widgets.
+- _Migration mode_: Use the React client and ignore incompatible widgets. Recommended for trying out the new client.
```
Comment (Robinfr) [inline content/en/docs/refguide/modeling/app-explorer/app/app-settings/_index.md]: Should we mention that we will display placeholders for incompatible widgets in this case?

File: content/en/docs/refguide/modeling/app-explorer/app/app-settings/_index.md [hunk 4]
```diff
@@ -27,19 +27,27 @@ For more information on settings in a configuration, see [Configuration](/refgui
 
 These settings influence the behavior of the Runtime when running your application.
 
-### 3.1 Static Resources from Disk
+### 3.1 Use React Client (Beta)
+
+This setting enables the new React client. This client was released into Beta with Mendix 10.7. There are three options:
+
+- _No_: Don't use the React (default).
```
Comment (Robinfr) [inline content/en/docs/refguide/modeling/app-explorer/app/app-settings/_index.md]: Meaning we use the Dojo client.

Also 
```suggestion
- _No_: Don't use the React client (default).
```

**Review by Robinfr — CHANGES_REQUESTED** [review]

File: content/en/docs/refguide/modeling/app-explorer/app/app-settings/_index.md (hunk 3 above)
Comment (ConnorLand) [reply content/en/docs/refguide/modeling/app-explorer/app/app-settings/_index.md]: Yes, I will add.

Comment (ConnorLand) [reply content/en/docs/refguide/modeling/app-explorer/app/app-settings/_index.md]: Looks good?

**Comment by ConnorLand** [comment]
> @EnasAbdelrazek we could, but would that add anything? We don't have for the other 10 out of 11 settings in this panel, and I don't think a picture of the radio buttons with exact labels as described will explain anything father.

True. Docs best practice is to not display screenshots of settings which are described in the doc's text. This best practice prevents duplication.

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**